docs/source/en/_toctree.yml

```diff
@@ -59,8 +59,8 @@
     sections:
     - local: tasks/image_classification
       title: Image classification
-    - local: tasks/semantic_segmentation
-      title: Semantic segmentation
+    - local: tasks/image_segmentation
```

Comment:
This will break old urls. In case we really want to rename the file, you can set up a redirect by doing something like https://github.com/huggingface/hub-docs/blob/main/docs/hub/_redirects.yml